compiler/ballerina-lang/src/test/java/io/ballerina/projects/test/resolution/packages/ResolutionWithNoDepsTomlTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.projects.test.resolution.packages;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * Package resolution test cases with no Dependencies.toml files.
+ */
+public class ResolutionWithNoDepsTomlTest extends AbstractPackageResolutionTest {
+
+    @Test(dataProvider = "resolutionTestCaseProvider")
+    public void testcase(String testSuite, String testCase, boolean sticky) {
+        runTestCase(testSuite, testCase, sticky);
+    }
+
+    @DataProvider(name = "resolutionTestCaseProvider")
+    public static Object[][] testCaseProvider() {
+        return new Object[][]{
```

Comment:
Can use something like
``` java
Files.newDirectoryStream(Paths.get(".."), "suite-*/case-*")
```
 to automate this. That way people who add tests don't need to touch java.

Reply:
How can we disable some tests in this case?

Reply:
Hmm I can think following :

```bash
mkdir disabled
mv suite-666/case-420 disabled
```